acis_thermal_check/state_builder.py

```diff
@@ -216,30 +222,233 @@ def get_validation_states(self, datestart, datestop):
 
         return states
 
+#-------------------------------------------------------------------------------
+# ACIS Ops Load History assembly 
+#-------------------------------------------------------------------------------
 class ACISStateBuilder(StateBuilder):
-    def __init__(self, logger=None):
-        raise NotImplementedError
 
-class HDF5StateBuilder(StateBuilder):
-    """
-    The HDF5StateBuilder obtains states from the commanded
-    states database using Chandra.cmd_states.fetch_states.
-    It can only be used for model validation and not prediction,
-    since this database is only updated after load products
-    are approved.
-    """
-    def get_prediction_states(self, tlm):
+    def __init__(self, interrupt=False, backstop_file=None, nlet_file = None, cmd_states_db="sybase", logger=None):
         """
-        Get the states used for the thermal prediction.
-        NOT IMPLEMENTED for HDF5StateBuilder.
+        Give the ACISStateBuilder arguments that were passed in 
+        from the command line, and set up the connection to the 
+        commanded states database. 
 
         Parameters
         ----------
-        tlm : dictionary
-            Dictionary containg temperature and other telemetry
+        interrupt : boolean
+            If True, this is an interrupt load.
+        backstop_file : string
+            Path to the backstop file. If a directory, the backstop 
+            file will be searched for within this directory.
+        nlet_file: full path to the Non-Load Event Tracking file
```

Comment:
We should make this item like the others in this docstring--the argument comes first separated from its type by a colon, then the description is on the next line indented.